src/components/navigation.tsx

```diff
@@ -136,78 +147,91 @@ export default function Navigation({
-                  <Disclosure.Button className="inline-flex items-center justify-center rounded-md bg-white p-2 text-gray-400 hover:bg-gray-100 hover:text-gray-500 focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2">
-                    <span className="sr-only">Open main menu</span>
-                    {open ? (
-                      <XMarkIcon className="block h-6 w-6" aria-hidden="true" />
-                    ) : (
-                      <Bars3Icon className="block h-6 w-6" aria-hidden="true" />
-                    )}
-                  </Disclosure.Button>
-                </div>
-              </div>
-            </div>
-
-            <Disclosure.Panel className="sm:hidden">
-              <div className="space-y-1 pt-2 pb-3">
-                {navigation.map(item => {
-                  const isCurrentPathname = item.href === pathname;
-                  return (
-                    <Disclosure.Button
-                      key={item.name}
-                      as={Link}
-                      href={item.href}
-                      className={classNames(
-                        isCurrentPathname
-                          ? 'bg-indigo-50 border-indigo-500 text-indigo-700'
-                          : 'border-transparent text-gray-600 hover:bg-gray-50 hover:border-gray-300 hover:text-gray-800',
-                        'block pl-3 pr-4 py-2 border-l-4 text-base font-medium'
+                  <div className="flex">
+                    <div className="hidden sm:-my-px sm:flex sm:space-x-8">
+                      {navigation.map(item => {
+                        const isCurrentPathname = item.href === pathname;
+                        return (
+                          <Link
+                            key={item.name}
+                            href={item.href}
+                            className={classNames(
+                              isCurrentPathname
+                                ? 'font-bold border-sky-700'
+                                : 'border-transparent hover:border-blue-300 hover:text-gray-700',
+                              'inline-flex items-center px-1 pt-1 border-b-2'
+                            )}
+                            aria-current={
+                              isCurrentPathname ? 'page' : undefined
+                            }
+                          >
+                            {item.name}
+                          </Link>
+                        );
+                      })}
+                    </div>
+                  </div>
+                  <div className="-mr-2 flex items-center sm:hidden ">
+                    {/* Mobile menu button */}
+                    <Disclosure.Button className="inline-flex items-center justify-center rounded-md bg-white p-2 text-gray-900 hover:bg-gray-100 hover:text-gray-500 focus:outline-none focus:ring-2 focus:ring-sky-600 focus:ring-offset-2">
+                      <span className="sr-only">Open main menu</span>
```

Comment:
Should we translate/localize this?

Reply:
Done. And I changed the text to "Open menu"